cfme/cloud/keypairs.py

```diff
@@ -128,8 +128,12 @@ class KeyPair(BaseEntity, WidgetasticTaggable):
 
     def delete(self, cancel=False, wait=False):
         view = navigate_to(self, 'Details')
-        view.toolbar.configuration.item_select('Remove this Key Pair',
-                                               handle_alert=(not cancel))
+        if self.appliance.version < "5.9":
```

Comment:
**Required**
I would rather see a single call to item_select, and construction of the message based on version before the call.

example of how I made a very similar change:
https://github.com/ManageIQ/integration_tests/pull/5815/files

Reply:
Ok will update the code.